.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -367,14 +372,15 @@ def write(x=''):
                 "capabilities": ["default"],
                 "address": "localhost",
                 "username": "",
-                "num_autograding_workers": NUM_GRADING_SCHEDULER_WORKERS
+                "num_autograding_workers": NUM_GRADING_SCHEDULER_WORKERS,
+                "enabled" : True
             }
         }
 
         with open(WORKERS_JSON, 'w') as workers_file:
             json.dump(worker_dict, workers_file, indent=4)
-    shutil.chown(WORKERS_JSON, 'root', HWCRON_GROUP)
-    os.chmod(WORKERS_JSON, 0o440)
+    shutil.chown(WORKERS_JSON, HWCRON_USER, 'root')
+    os.chmod(WORKERS_JSON, 0o640)
```

Comment:
why are we changing the permissions for the file?

Reply:
hwcron now needs write access, as it is able to update the 'enabled' field, turning off workers as necessary.